contribute/developer-guide.md

```diff
@@ -95,7 +97,7 @@ You can build the back-end as follows:
 2. Generate code using Wire:
 
 ```
-# Normally Wire tool installed at $GOPATH/bin/wire.exe
+# Normal Wire tool install path: $GOPATH/bin/wire.exe
```

Comment:
```suggestion
# Wire tool install path: $GOPATH/bin/wire.exe
```
```suggestion
# Normal Wire tool install path: $GOPATH/bin/wire.exe
```

Reply:
Maybe "default" is a better word than "normal" here -- I'm guessing this is trying to say "it's usually here, but you may have set it up differently on your system", basically